docs/pages/guides/environment-variables.mdx

```diff
@@ -59,16 +61,31 @@ If you create an environment-specific file, like **.env.test**, you can load it
 
 <Terminal cmd={['$ NODE_ENV=production npx expo start']} />
 
+### Disabling environment variables
+
+Environment variables in Expo CLI have two parts, both can be disabled:
+
+1. Expo CLI will load the `.env` files into the global process. This can be disabled by setting the environment variable `EXPO_NO_DOTENV` before any Expo CLI command is run, like `EXPO_NO_DOTENV=1`.
```

Comment:
```suggestion
1. Expo CLI automatically loads the **.env** files into the global process. To disable this behavior, set the environment variable `EXPO_NO_DOTENV` to `1` before running any Expo CLI command: `EXPO_NO_DOTENV=1`.
```

- [Filenames to be bold as per writing styleguide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#filenames-as-bold-text)
- [Using present tense](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#use-present-tense)
- Previously, it was suggested: "...by setting the environment variable `EXPO_NO_DOTENV`". This was missing as to what the env var needs to be set to or if just setting the env var is enough.